source/server/server.cc

```diff
@@ -15,6 +15,7 @@
 #include "envoy/upstream/cluster_manager.h"
 
 #include "common/api/api_impl.h"
+#include "common/common/backward.h"
```

Comment:
As we came to the conclusion IRL, we want the backward dependency to be in only a few files that can be easily excluded from the build. Maybe we can add an abstract `backtrace` interface to `include/envoy` and have folks only depend on that. This would include APIs to dump stacktrace. Then, in `envoy_build_system.bzl`, when setting up the `deps` for things like `envoy_cc_library`, we can point it at the specific implementation that uses Backwards. For Google's (or anyone else who wants to override backwards) import, they can replace this dependency with their own.

@mattklein123 How do you override the signal handlers in the Lyft build? We would like to be able to wrap `main()` with our own extras for initialization (and others might want to as well).